include/openenclave/internal/sgx/plugin.h

```diff
@@ -72,9 +73,9 @@ oe_result_t oe_sgx_extract_claims(
  * @retval OE_INVALID_PARAMETER At least one parameter is invalid.
  * @retval An appropriate error code on failure.
  */
-oe_result_t oe_sgx_hash_custom_claims(
-    const void* custom_claims,
-    size_t custom_claims_size,
+oe_result_t oe_sgx_hash_custom_claims_buffer(
```

Comment:
Please rename parameter description at line 68 and 69.

Reply:
Fixed